resources/zib/zib-Vaccination-event.xml

```diff
@@ -167,5 +168,9 @@
         <comment value="Comment" />
       </mapping>
     </element>
+    <element id="Immunization.protocolApplied.doseNumber[x]">

```

Comment:
Ahh ... this comment should be applied in the nl-core layer (because that's where you introduce the co-requirement to specify this concept).

Reply:
I moved the comment